examples/nlp/language_modeling/megatron_retro_mutransfer_pretrain.py

```diff
@@ -1,83 +0,0 @@
-# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
why we delete the mu-transfer pretraining?

Reply:
Regard this,
I didn't mean to remove the current RETRO NeMo implementation and features. Locally I changed them to `*_legacy.py`. In this PR, I didn't put in all the RETRO `*_legacy` files to avoid distraction. I will put them back to not cause confusion.

Besides that, we indeed need some discussion about putting NeMo RETRO and mcore RETRO. I'll start a Slack thread.